File: js/challenge0020/challenge0020.pl
sub lessThan100{
	my($x, $y) = @_;
	$x + $y < 100 ? "true" : "false";
}

print lessThan100(22, 15);
print "\n";
print lessThan100(83, 34);
print "\n";
print lessThan100(3, 77); 

File: js/challenge0020/challenge0020.c
#include <stdio.h>
#include <stdbool.h>

bool lessThan100(const int x, const int y) {
    return (x + y) < 100;
}

int main(){
    printf("%d\n", lessThan100(22, 15));
    printf("%d\n", lessThan100(83, 34));
    printf("%d\n", lessThan100(3, 77));
}
